### DISCUSSION TITLE
Explicity state a recommended installation option to avoid confusion

### DISCUSSION THREAD

**Discussion by ahtcx** [pnpm/pnpm#4013] [General]
Right now the install page provides many different ways to install pnpm, which can lead to confusion (especially with newer developers) as it's not clear what advantages (if any) one method would have over the over.

It would be nice to have a big main command that just works for 99%, whilst keeping the other options as more of a "I have a weird system maybe one of these options will help me" thing.

I also think that command should be on the homepage of the website, I love pnpm and would love to make it as friction-less as possible to get it widely adopted 😁

**Comment by zkochan** [comment]
I would probably recommend the standalone bash script but the easiest one is probably using corepack
